src/cascadia/TerminalApp/defaults.json

```diff
@@ -276,6 +276,7 @@
         { "command": "toggleFullscreen", "keys": "f11" },
         { "command": "openNewTabDropdown", "keys": "ctrl+shift+space" },
         { "command": "openSettings", "keys": "ctrl+," },

```

Comment:
The regular `openSettings` command here was left untouched because when Settings UI comes out, we'll sneakily change the default behavior from `target: settingsFile` to `target: UI`.